lib/active_model_serializers/adapter/json_api/relationship.rb

```diff
@@ -6,21 +6,22 @@ class Relationship
         # {http://jsonapi.org/format/#document-links Document Links}
         # {http://jsonapi.org/format/#document-resource-object-linkage Document Resource Relationship Linkage}
         # {http://jsonapi.org/format/#document-meta Docment Meta}
-        def initialize(parent_serializer, serializer, options = {}, links = {}, meta = nil)
+        def initialize(parent_serializer, serializer, serializable_resource_options, args = {})
```

Comment:
should we start naming options like this all over the place? I think it would help readability a ton, since we have so many different kinds of options for different purposes.